src/app/api/userAuth.ts

```
@@ -0,0 +1,71 @@
+	console.log(data);
+};
+
+/* 로그인 반환 예시
+  "teamId": "7",
+  "userId": 1122,
+  "iat": 1739323087,
+  "exp": 1739326687
+*/
+
+// 회원가입 기능
+const signupUser = async ({
```

ESLint

[error] 'signupUser' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)